sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwapWithNamedConventionsExamples.cs

```diff
@@ -60,5 +60,12 @@ public void InterestRateSwapWithNamedConventionsCreationAndUpsertionExample()
             // Delete Instrument
             _instrumentsApi.DeleteInstrument("ClientInternal", uniqueId); 
         }
+        
+        [TestCase(ModelSelection.ModelEnum.SimpleStatic)]
+        public void InterestRateSwapValuationExample(ModelSelection.ModelEnum model)
```

Comment:
How is this function different than the function you added to InterestRateSwapExamples.cs?

Reply:
Copy, paste and forgot to edit - whoops!